README.md

```diff
@@ -3,23 +3,19 @@
 <img align="left" src="https://avatars.githubusercontent.com/u/125200832?s=128&v=4" alt="ratatui logo">
 
 This is the documentation project for [Ratatui](https://github.com/tui-rs-revival/ratatui), a Rust
-library to build rich terminal user interfaces and dashboards. The documentation can be read at
-<https://tui-rs-revival.github.io/ratatui-book/>.
+library to build rich terminal user interfaces and dashboards. Note that this documentation is supplementary
+to the technical documentation, and we also maintain crate documentation. This is more of a guide book
```

Comment:
Isn't the technical documentation the same as the crate documentation ?

Reply:
They are the same, it's just poorly-worded. I meant "the technical documentation is at crates.io" basically.